### DISCUSSION TITLE
CLIP Sentence Transformer

### DISCUSSION THREAD

**Discussion by wrmay** [deepjavalibrary/djl#3293] [Q&A] [answered]
I'm new here.  I'm interested in creating CLIP based embeddings in Java.  Is a CLIP sentence transformer available ?

**Comment by frankfliu** [comment, marked as answer]
Please take a look this example: https://github.com/deepjavalibrary/djl/blob/master/examples/src/main/java/ai/djl/examples/inference/clip/ClipModel.java#L52

**Reply by wrmay** [reply]
Thank you!
